packages/components/src/sandbox/types.ts

```diff
@@ -31,4 +31,10 @@ export type SandBoxProps = {
 	 * The `onFocus` callback for the iframe.
 	 */
 	onFocus?: React.DOMAttributes< HTMLIFrameElement >[ 'onFocus' ];
+	/**
+	 * * The `tabindex` the iframe should receive.
+	 *
+	 * @default 0
+	 */
+	tabIndex?: number;
```

Comment:
Optional prop and number type? I think this is okay.

Reply:
Even better:

``` suggestion
	tabIndex?: HTMLElement[ 'tabIndex' ]
```

Reply:
Great, thanks.